TestCases/hom_euler/NACA0012_5thOrder/fem_NACA0012.cfg

```diff
@@ -185,7 +185,6 @@ SURFACE_ADJ_FILENAME= surface_adjoint
 OUTPUT_WRT_FREQ= 50000
 %
 % Writing convergence history frequency
-WRT_CON_FREQ= 1
 %
```

Comment:
I don't think it was "third time lucky" xD
We should probably squash-merge to compress all these attempts.

Reply:
shoot I forgot the text above 🤦 
Sure I am fine with squash merge. The upside of doing it as usual would be easy reverting of single option deprecations ... but I am not nostalgic towards any of these options, so lets squash it  🎾

Reply:
99% of what we removed was not working, I don't think we need to bring them back.